panther_analysis_tool/main.py

```diff
@@ -100,6 +109,14 @@ def __init__(self, analysis_id: str):
         super().__init__(self.message)
 
 
+# exception for conflicting ids
+class AnalysisContainsDuplicatesException(Exception):
+    def __init__(self, analysis_id: str, invalid_fields: List[str]):
+        self.message = "Specification with duplicates AnalysisID: [{}] : [{}]" \
+            .format(analysis_id, ', '.join(x for x in invalid_fields))
+        super().__init__(self.message)
```

Comment:
I love the custom error message! Although I think this error message is not 100% clear to someone not very familiar with Panther, I think maybe:

```
"Specification file for [{}] contains fields with duplicate values: [{}]".format(analysis_id, ', '.join(x for x in invalid_field)
```

might better capture the meaning. What do you think?

Reply:
That's definitely more informative and less ambiguous